src/Symfony/Component/Form/Extension/DataCollector/FormDataCollector.php

```diff
@@ -75,6 +75,10 @@ public function __construct(FormDataExtractorInterface $dataExtractor)
 
     /**
      * Does nothing. The data is collected during the form event listeners.
+     *
+     * @param Request         $request
+     * @param Response        $response
+     * @param \Exception|null $exception
```

Comment:
Please, remove those new phpdocs (and almost every others you added). We don't add phpdocs when it does not add something that cannot be already inferred by the signature.